src/Client.ts

```diff
@@ -230,6 +220,21 @@ export default class Client {
     return this.slamServiceClient
   }
 
+  get serviceConnection () {
+    const clientTransportFactory = this.sessionOptions?.disabled
+      ? this.transportFactory
+      : this.sessionManager.transportFactory
+
+    if (!clientTransportFactory) {
+      throw new Error(Client.notConnectedYetStr)
+    }
+    const grpcOptions = { transport: clientTransportFactory }
+    return {
+      grpcOptions,
+      serviceHost: this.serviceHost,
+    }
+  }
+
```

Comment:
this exposes connection parameters so that component service clients can instantiate themselves given a `Client`, rather than making the `Client` responsible for instantiating each component client itself.